src/crimson/os/seastore/transaction_manager.h

```diff
@@ -178,8 +178,14 @@ class TransactionManager : public ExtentCallbackInterface {
   {
     auto v = pin->get_logical_extent(t);
```

Comment:
`Cache::get_extent_viewable_by_trans()` should be patched to make sure user can only see fully loaded extents.